src/components/ui/Collapsible.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState, ReactElement } from "react";
+import ArrowRight from "../../assets/icons/arrow-right.svg";
+import ArrowDown from "../../assets/icons/arrow-down.svg";
+
+type CollapsibleProps = React.PropsWithChildren<{
+  height?: number;
+  width?: number;
+  children: ReactElement;
+}>;
+
+const Collapsible = ({
+  height,
+  width,
+  children,
+}: CollapsibleProps): ReactElement => {
+  const Open = () => <ArrowDown width={height} height={width}></ArrowDown>;
+  const Closed = () => <ArrowRight width={height} height={width}></ArrowRight>;
+  const [open, setOpen] = useState(false);
+  const [icon, setIcon] = useState(<Closed />);
+  const toggle = () => {
+    const newIcon = !open ? <Open /> : <Closed />;
+    setIcon(newIcon);
+    setOpen(!open);
+  };
+
+  return (
+    <div onClick={() => toggle()}>
+      {icon}
+      {open && <div className="toggle">{children}</div>}
+    </div>
+  );
+};
+
+Collapsible.defaultProps = {
+  height: 24,
+  weight: 24,
+};
```

Comment:
Nit: since we're never giving these as arguments to `Collapsible`, I'd just make them constants and use them in `ArrowDown` and `ArrowRight`. This way, the reader doesn't have to guess about the impact of changing those values, or question themselves of why these are arguments.

Reply:
its funny. originally I had it that way and then expanded it because I was future-predicting and thought it would be useful to have the arguments already in place. Happy to revert.

Reply:
Crystal balls are often wrong, the ease of adding said arguments when we need them, and the risk of letting something think "maybe there's a great reason for this I'm not seeing", all of them make me not want to have those arguments. :)